compose/const.py

```diff
@@ -74,4 +76,5 @@
     API_VERSIONS[COMPOSEFILE_V3_5]: '17.06.0',
     API_VERSIONS[COMPOSEFILE_V3_6]: '18.02.0',
     API_VERSIONS[COMPOSEFILE_V3_7]: '18.06.0',
+    API_VERSIONS[COMPOSEFILE_V3_8]: '19.03.0',
```

Comment:
as both `COMPOSEFILE_V3_7` and `COMPOSEFILE_V3_8` == `'1.38'` doesn't this override `API_VERSIONS['1.28']` just being set to `'18.06.0'` ?

Reply:
Yep, but I guess this mapping is more for us to easily view the minimum version needed for each compose file.